snap/snapcraft.yaml

```diff
@@ -743,6 +743,9 @@ parts:
       (
         cd $GOPATH/src/github.com/opencontainers/runc
         git checkout ${RUNC_COMMIT}
+        git config user.email "microk8s-builder-bot@ubuntu.com"
+        git config user.name "MicroK8s builder bot"
```

Comment:
I don't recall this being necessary, can you not just do `git apply ...` ? I suppose that gives you a dirty tree, does the makefile for runc not like building dirty trees?